pom.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.greenbird.utilihive.stubs</groupId>
+    <artifactId>utilihive-backend-stub-sftp</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Utilihive SFTP Backend Stub</name>
+    <description>Stub for SFTP backend, for use in functional testing of Utilihive integration flows</description>
+    <url>https://github.com/utilihive/utilihive-backend-stub-sftp-server</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://github.com/utilihive/utilihive-backend-stub-sftp-server/blob/main/LICENSE</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Greenbird Integration Technology</name>
+            <email>operations@greenbird.com</email>
+            <organization>Greenbird Integration Technology</organization>
+            <organizationUrl>https://www.greenbird.com</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/utilihive/utilihive-backend-stub-sftp-server.git</connection>
+        <developerConnection>scm:git:ssh://github.com:utilihive/utilihive-backend-stub-sftp-server.git</developerConnection>
+        <url>https://github.com/utilihive/utilihive-backend-stub-sftp-server/tree/main</url>
+    </scm>
+
+    <properties>
```

Comment:
Lets keep the version properties in alphabetical order and add a blank line between them and the project property,

Reply:
Reordered and added all the versions as properties